doc/kube-setup-data-ingestion-job.md

```diff
@@ -0,0 +1,16 @@
+The data-ingestion-job is specific to DataSTAGE.
+
+To prep: 
```

Comment:
I think we need an additional prep line about adding the ingestion image to your `manifest.json` for the commons and perhaps an example. Otherwise you get a sort of cryptic error when trying to run